src/integration-test/java/se/sundsvall/casedata/apptest/ErrandListenerIT.java

```diff
@@ -43,6 +43,7 @@ void test01_persistErrandUnknown() {
 		setupCall()
 			.withHttpMethod(POST)
 			.withServicePath(getPath())
+			.withHeader(AD_USER_HEADER_KEY, "user123")
```

Comment:
Should not this test case have ad-user "UNKNOWN" as it is when no jwt header is found?